package.json

```diff
@@ -17,11 +17,12 @@
   "homepage": "https://github.com/firstandthird/eslint-config-firstandthird#readme",
   "devDependencies": {
     "chai": "^3.5.0",
+    "mocha": "^3.0.2",
     "eslint": "^3.4.0",
-    "mocha": "^3.0.2"
+    "eslint-plugin-import": "^1.14.0"
   },
-  "peerDependencies": {
+  "dependencies": {
```

Comment:
It should be peerDependencies.  But I think we then also need to put eslint/plugin-import in devDependencies, since Travis doesn't seem to install peer dependencies and our tests aren't passing there.   Can I put those in both peerDependencies and devDependencies, or do you have another way?